.kitchen.yml

```diff
@@ -33,6 +33,17 @@ platforms:
       run_command: /sbin/init
       volume:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  # CentOS7 with Systemd
+  - name: centos-7
+    driver_config:
+      platform: centos
+      dockerfile: .kitchen-docker/centos7/Dockerfile
+      # TODO: Use non-root user after updating upstream Docker image: https://github.com/StackStorm/st2-dockerfiles/issues/38
+      username: root
+      password: docker.io
```

Comment:
As a workaround for CentOS7 we use different approach comparing to CentOS6 #81

As @Mierdin discovered before non-root login is blocked on current CentOS7 image. But we still can use root.

Additionally, I created [CentOS7 packagingtest needs re-build #38](https://github.com/StackStorm/st2-dockerfiles/issues/38) with detailed description so we fix it in future.